NuRadioReco/framework/base_station.py

```diff
@@ -38,6 +58,8 @@ def get_parameter(self, key):
         if not isinstance(key, parameters.stationParameters):
             logger.error("parameter key needs to be of type NuRadioReco.framework.parameters.stationParameters")
             raise ValueError("parameter key needs to be of type NuRadioReco.framework.parameters.stationParameters")
```

Comment:
I don't agree with this change. We still want to reconstruct these properties on station level, so we need to store them on station level! Otherwise we can't reconstruct an event with multiple stations anymore! Only modules that use all stations should write their reconstruction into the event or shower class. 

For simulated events this might be different, as we know the one true shower properties.